docs/source/tutorial/dynamodb.rst

```diff
@@ -0,0 +1,293 @@
+.. _dynamodb_tutorial
+
+DynamoDB Tutorial
+=================
+This tutorial shows how to use
+`Amazon DynamoDB <http://aws.amazon.com/documentation/dynamodb/>`_ with Boto3.
+It will focus on how to use the :py:class:`DynamoDB.ServiceResource` and
+:py:class:`DynamoDB.Table` resources.
+
+
+Creating a New Table
+--------------------
+
+In order to create a new table, use the
+:py:meth:`DynamoDB.ServiceResource.create_table` method::
+
+    import boto3
+    
+    # Get the service resource.
+    dynamodb = boto3.resource('dynamodb')
+
+    # Create the DynamoDB table.
+    table = dynamodb.create_table(
+        TableName='users',
+        KeySchema=[
+            {
+                'AttributeName': 'username',
+                'KeyType': 'HASH'
+            },
+            {
+                'AttributeName': 'last_name',
+                'KeyType': 'RANGE'
+            }
+        ],
+        AttributeDefinitions=[
+            {
+                'AttributeName': 'username',
+                'AttributeType': 'S'
+            },
+            {
+                'AttributeName': 'last_name',
+                'AttributeType': 'S'
+            },
+
+        ],
+        ProvisionedThroughput={
+            'ReadCapacityUnits': 5,
+            'WriteCapacityUnits': 5
+        }
+    )
+
+    # Wait until the table exists.
+    table.meta.client.get_waiter('table_exists').wait(TableName='users')
```

Comment:
Didn't realize the waiters weren't mapped in the ddb resource model.  Can we get this updated?  This is an awkward thing to make customers write.